scripts/common/object_store.sh

```diff
@@ -117,6 +123,11 @@ EOF
     spinner_until 300 kubernetes_pod_completed sync-object-store default || true
     kubectl logs -f sync-object-store || true
 
+    # even if the migration failed, we should ensure ekco is running again
+    if kubernetes_resource_exists kurl deployment ekc-operator; then
+        kubectl -n kurl scale deploy ekc-operator --replicas=1
+    fi
+
```

Comment:
Wdyt about check it is running again before move forward ?

Reply:
there's no need - we don't use it during the minio migration process (or generally any of the kurl processes)

it's more for all the cluster management stuff that comes up during times that we're not running an installer, if that makes any sense